modules/promotions/module.php

```diff
@@ -112,6 +115,27 @@ public function enqueue_react_data(): void {
 		);
 	}
 
+	public function enqueue_editor_v4_alphachip(): void {
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return;
+		}
+
+		$min_suffix = Utils::is_script_debug() ? '' : '.min';
+
+		wp_enqueue_script(
+			'editor-v4-opt-in-alphachip',
+			ELEMENTOR_ASSETS_URL . 'js/editor-v4-opt-in-alphachip' . $min_suffix . '.js',
```

Comment:
This was the main aspect of this PR. Now that we can enqueue the Alphachip independently, we can handle the expected dependencies as well. Now it will work regardless of which experiments have been activated.